models/inventory-entry/src/inventory-entry-draft/presets/sample-data-b2b/sku-a789-bc-2015-default.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  ChannelDraft,
+  type TChannelDraft,
+} from '@commercetools-test-data/channel';
+import {
+  ProductVariantDraft,
+  type TProductVariantDraft,
+} from '@commercetools-test-data/product';
+import type { TInventoryEntryDraftBuilder } from '../../../types';
+import * as InventoryEntryDraft from '../../index';
+
+const defaultWarehouse = ChannelDraft.presets.sampleDataB2B
+  .defaultWarehouse()
+  .build<TChannelDraft>();
+
+const a789Bc01 = ProductVariantDraft.presets.sampleDataB2B
+  .a789Bc01()
+  .build<TProductVariantDraft>();
+
+const skuA789Bc2015Default = (): TInventoryEntryDraftBuilder =>
+  InventoryEntryDraft.presets
+    .empty()
+    .key(`${a789Bc01.sku}-${defaultWarehouse.key}`)
+    .sku(a789Bc01.sku!)
+    .quantityOnStock(1000)
+    .channel({ typeId: 'channel', key: defaultWarehouse.key });
```

Comment:
These should be `KeyReferenceDraft.presets.channel().key(defaultWarehouse.key)`. And like the linter pointed out, `channel` doesn't exist, perhaps this should be `supplyChannel`?

Reply:
Absolutely right. I updated the files